packages/tadviewer/src/components/SelectionChangeData.tsx

```diff
@@ -0,0 +1,16 @@
+export interface SelectionChangeData {
+  readonly value: any;
```

Comment:
Similar to earlier comments -- a single `value` and single `column` here seem overly tailored to the special case of rectangular selection that is just a single cell.